src/blocks/battery.rs

```diff
@@ -24,6 +23,7 @@
 //!
 //! Placeholder  | Value                                                                   | Type              | Unit
 //! -------------|-------------------------------------------------------------------------|-------------------|-----
+//! `icon`       | A static icon                                                           | Icon   | -
```

Comment:
This looks confusing and wrong: this icon is not "static" but it represents the state of the battery. Same applies for some other blocks.

Reply:
Uhh, I understood the description differently, more along the lines "a hard reference in code to the icon, see "used icons"" below. I'll improve it